### PR TITLE
Add & Fix Chinese Translation

### PR DESCRIPTION
**Pull request by Bill-Haku**
Translation includes:
- Inazuma and other countries in `characters.json5` (All done.).
- All of `items.json5`.

Fix errors includes:
- Wrong translation of Dr. Livingstone & Dr. Edith in `characters.json5`.
- Wrong translation of Moment of Syzygy in `domains.json5`.

### CODE REVIEW

**Comment by xicri** [comment]
@Bill-Haku Thanks, send me the review request when you are ready.

**Comment by Bill-Haku** [comment]
@xicri Everything is ready. You can review it now.

File: dataset/dictionary/items.json5 [hunk 1]
```diff
@@ -2126,54 +2126,67 @@
...
   },
   {
     en: "Crystalfish",
     ja: "水晶宴",
+    zhCN: "水晶宴",
     tags: [ "item" ],
   },
+  // Note: seems missing Dawncatcher, a type of Medaka that can be found as a pet for the sereniteapot
+  // Also, many other types of fishes are missing, you may need to complete them later.
+  // zhCN: "擒霞客"
```
Comment (xicri) [inline dataset/dictionary/items.json5:2159]: Unfortunately, I don't play fishing so much, so I don't have so many kinds of fish. That's why there are many missing fishes.
You can add the words if you know the Chinese-English translations of fish. Japanese translation is not the required parameter.

```suggestion
  {
    en: "Dawncatcher",
    zhCN: "擒霞客",
    tags: [ "item" ],
  },
```

File: dataset/dictionary/items.json5 [hunk 2]
```diff
@@ -2235,74 +2235,87 @@
...
     en: "Sangonomiya Chronicles",
     ja: "珊瑚宮記",
+    zhCN: "珊瑚宫记",
     pronunciationJa: "さんごのみやき",
     tags: [ "item", "inazuma" ],
   },
   {
     en: "Yakshas: The Guardian Adepti",
     ja: "仙衆夜叉 / 護法仙衆夜叉録",
+    zhCN: "护法仙众夜叉录",
```
Comment (xicri) [inline dataset/dictionary/items.json5:2284]: In Japanese, "Yakshas: The Guardian Adepti" has two meanings.
1. 仙衆夜叉: Group of the Yakshas which 魈, 浮舍, 弥怒, 伐难, and 应达 belonged to.
2. 護法仙衆夜叉録: a book written about the Xiao and other Yakshas.

I guess 护法仙众夜叉录 is translation for 2, and I guess there is also Chinese translation for 1.

File: dataset/dictionary/characters.json5 [hunk 3]
```diff
@@ -1680,7 +1680,7 @@
   {
     en: "Nameless",
     ja: "無名",
-    zhCN: "无名",  
+    zhCN: "无名",
```
Comment (xicri) [inline dataset/dictionary/characters.json5:1683]: Just FYI, if your editor has EditorConfig plugin and install it, the trailing spaces are automatically deleted on save.
Ignore this if you already know that.

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/characters.json5 [hunk 4]
```diff
@@ -1891,18 +1908,21 @@
   {
     en: "Pulcinella",
     ja: "プルチネラ",
+    zhCN: "「公鸡」普契涅拉",
```
[the remark this answers is not in the document]
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:1910]: You can refer to this [wiki](https://genshin-impact.fandom.com/wiki/Pulcinella). And you can see this title seems different between English and Japanese or Chinese.

File: dataset/dictionary/items.json5 [hunk 5]
```diff
@@ -1092,170 +1092,208 @@
...
   {
     en: "Debris of Decarabian's City",
     ja: "高塔の王の残壁",
+    zhCN: "高塔孤王的残垣",
     pronunciationJa: "こうとうのおうのざんぺき",
     tags: [ "item" ],
   },
+  // Note: no Tile of Decarabian's Tower? (zhCN: "高塔孤王的破瓦")
+  // Seems that all the corresponding 2star items following until items form Inazuma are missing.
+  // I will leave the Chinese translation in the place by order.
```
[the remark this answers is not in the document]
Comment (Bill-Haku) [reply dataset/dictionary/items.json5]: OK

File: dataset/dictionary/items.json5 (hunk 1 above)
Comment (Bill-Haku) [reply dataset/dictionary/items.json5]: Okay, I will do it after this PR

File: dataset/dictionary/items.json5 (hunk 2 above)
Comment (Bill-Haku) [reply dataset/dictionary/items.json5:2286]: As you put it in the place of books, so I copied the title of the book from the game.

仙众夜叉 is the translation for 1, I will make a commit for it.

File: dataset/dictionary/characters.json5 (hunk 4 above)
[the remark this answers is not in the document]
Comment (xicri) [reply dataset/dictionary/characters.json5:1910]: Thanks, it seems "雄鶏" in Japanese and "Rooster" in English. I'll add it.

File: dataset/dictionary/characters.json5 (hunk 3 above)
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:1683]: Thanks for your advice. The plugin is helpful.

File: dataset/dictionary/items.json5 (hunk 2 above)
Comment (xicri) [reply dataset/dictionary/items.json5]: > As you put it in the place of books, so I copied the title of the book from the game.

Year, sorry for confusion.

File: dataset/dictionary/characters.json5 (hunk 4 above)
[the remark this answers is not in the document]
Comment (xicri) [reply dataset/dictionary/characters.json5:1910]:
```suggestion
    zhCN: "普契涅拉",
```

File: dataset/dictionary/characters.json5 [hunk 6]
```diff
@@ -1891,18 +1908,21 @@
...
     tags: [ "snezhnaya", "character-sub" ],
   },
   {
```
Comment (xicri) [inline dataset/dictionary/characters.json5:1913]:
```suggestion
  {
    en: "雄鶏",
    ja: "Rooster",
    zhCN: "公鸡",
    tags: [ "snezhnaya", "character-sub", "title" ],
  },
  {
```

**Review by xicri — APPROVED** [review]

**Review by xicri — CHANGES_REQUESTED** [review]
Oops, I accidentally approved.

File: dataset/dictionary/characters.json5 (hunk 6 above)
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:1913]: It seems that you made a mistake for en and ja content.

Comment (xicri) [reply dataset/dictionary/characters.json5:1913]: Oops.
I've fixed it.

**Review by xicri — DISMISSED** [review]
Thanks, LGTM!

BTW, why don't we release this PR tomorrow?
Since we will have v2.8 announcement program tonight, we can get less attention from the audience if we released today.

**Comment by Bill-Haku** [comment]
That makes a sense. We can postpone releasing it.

**Review by xicri — APPROVED** [review]

**Pull request merged**